src/OrchardCore.Modules/OrchardCore.Navigation/PagerShapes.cs

```diff
@@ -339,6 +340,15 @@ public Task<IHtmlContent> Pager(Shape Shape, dynamic DisplayAsync)
 
             var routeData = new RouteValueDictionary(Html.ViewContext.RouteData.Values);
 
+            //Allows to pass custom url params to PagerSlim
```

Comment:
Proposition from @jtkech here is to use HttpContext.Request.Querystring and HttpContext.Request.Form arguements instead and pass them all.

Reply:
Question is should we always pass all Querystring params to the Pagers or should we restrict to only what we want?